src/components/Navbar.tsx

```diff
@@ -12,22 +11,8 @@ export default function Navbar() {
.9 0 17.9-11.8 17.9-26.4zm75.6-26.4c-3.8 0-7.8.6-10.1 1.8 1.9 2.2 4.5 7.3 4.5 13.4 0 13.1-10.8 23.2-24.1 23.2-5.3 0-8.1-.8-11.7-1.8-3.6 1-5.2 3.2-5.2 6 0 3.1 2.1 4.9 5.4 4.9 31.3 0 41.4 5.6 41.4 18.5 0 9.7-9.5 17.8-30.2 17.8-18.5 0-28.5-5.1-28.5-16 0-6.6 4.1-9.8 11.3-13.4-5.1-1.6-8.3-5.5-8.3-9.4 0-5.8 3.8-8.2 8.9-11.4-6.2-4.7-9-11.1-9-18.4 0-14 10.9-23.1 25-23.1 7.1 0 12.1 1.5 16.4 4.6 3.9-2.8 9.3-5.4 14.2-5.9zm-31.4 56.7c-14.3 0-18.4 6.3-18.4 11 0 5 3.1 8.3 21.4 8.3 10.9 0 19.8-3.1 19.9-9.6-.1-6.4-6.7-9.7-22.9-9.7zm17.2-41.4c0-8.5-7.3-15.3-16.4-15.3s-16.4 6.8-16.4 15.3c0 8.4 7.3 15.2 16.4 15.2s16.4-6.8 16.4-15.2zm27.3-48.8h9.2v10.9h-9.2zm0 26.9h9.2v66h-9.2zm76 52c-4.4 7.5-10.5 15.4-25.5 15.4-14.7 0-28.7-10.2-28.7-34.3 0-21.4 10.7-34.4 26.9-34.4 13.8 0 25.2 9 25.2 30v6.1h-43c0 14.5 8.7 24.7 19.6 24.7 8.2 0 14.5-4.3 18.3-11.2zm-11.2-25c0-12.2-5.8-20.3-15.7-20.3-11.1 0-17 8.1-17.8 20.3zm27.1 22.3c4.3 6.3 10.6 10.2 18.9 10.2 9.9 0 14.2-5.3 14.2-10.9 0-16.8-39.1-7.9-38.4-32 .4-11.5 9.2-18 21.9-17.8 8.7.1 18.2 4.8 22.6 11.5l-6.3 5.6c-5.1-6.2-10-9.2-17.1-9.2-7.7 0-11.9 3.9-11.9 9.7 0 14.4 38.4 7 38.4 31.7 0 12.8-11.2 19.3-23.2 19.3-10.3 0-20.3-4.5-26.4-12.6z" fill="#939b50" fill-rule="evenodd" /><path d="m41.5 224.4c-22.9 0-41.5-18.5-41.5-41.5 0-22.9 18.6-41.5 41.5-41.5 23 0 41.6 18.6 41.6 41.5 0 23-18.6 41.5-41.6 41.5zm70.8 70.7c-23 0-41.6-18.5-41.6-41.5s18.6-41.5 41.6-41.5c22.9 0 41.5 18.5 41.5 41.5s-18.6 41.5-41.5 41.5zm70.7 70.7c-23 0-41.5-18.6-41.5-41.5 0-23 18.5-41.5 41.5-41.5s41.6 18.5 41.6 41.5c0 22.9-18.6 41.5-41.6 41.5zm0 141.3c-23 0-41.5-18.5-41.5-41.5 0-22.9 18.5-41.5 41.5-41.5s41.6 18.6 41.6 41.5c0 23-18.6 41.5-41.6 41.5zm70.7-70.6c-22.9 0-41.5-18.6-41.5-41.6 0-22.9 18.6-41.5 41.5-41.5 23 0 41.6 18.6 41.6 41.5 0 23-18.6 41.6-41.6 41.6zm70.8-70.7c-23 0-41.6-18.6-41.6-41.5 0-23 18.6-41.5 41.6-41.5 22.9 0 41.5 18.5 41.5 41.5 0 22.9-18.6 41.5-41.5 41.5z" fill="#4364a0" /><g fill="#939b50"><path d="m183 224.4c-23 0-41.5-18.5-41.5-41.5 0-22.9 18.5-41.5 41.5-41.5s41.6 18.6 41.6 41.5c0 23-18.6 41.5-41.6 41.5zm70.7-70.6c-22.9 0-41.5-18.6-41.5-41.6 0-22.9 18.6-41.5 41.5-41.5 23 0 41.6 18.6 41.6 41.5 0 23-18.6 41.6-41.6 41.6zm70.8 70.6c-23 0-41.6-18.5-41.6-41.5 0-22.9 18.6-41.5 41.6-41.5 22.9 0 41.5 18.6 41.5 41.5 0 23-18.6 41.5-41.5 41.5zm70.7 70.7c-23 0-41.6-18.5-41.6-41.5s18.6-41.5 41.6-41.5 41.6 18.5 41.6 41.5-18.6 41.5-41.6 41.5zm70.7 70.7c-23 0-41.5-18.6-41.5-41.5 0-23 18.5-41.5 41.5-41.5s41.6 18.5 41.6 41.5c0 22.9-18.6 41.5-41.6 41.5zm-141.4-282.7c-23 0-41.6-18.6-41.6-41.6 0-22.9 18.6-41.5 41.6-41.5 22.9 0 41.5 18.6 41.5 41.5 0 23-18.6 41.6-41.5 41.6z" /><path d="m396.7 388.1c-9.1 0-16.4-7.3-16.4-16.3 0-9.1 7.3-16.4 16.4-16.4s16.4 7.3 16.4 16.4c0 9-7.3 16.3-16.4 16.3zm13.7-16.3c0-7.9-6.1-14.2-13.7-14.2s-13.7 6.3-13.7 14.2c0 7.8 6.1 14.2 13.7 14.2s13.7-6.4 13.7-14.2zm-17.1 9.7h-2.7v-19.5h7.5c4.3 0 6.3 1.9 6.3 5.4s-2.2 5-4.5 5.4l5.4 8.7h-3l-5.3-8.7h-3.7zm3-10.8c2.6 0 5.4.1 5.4-3.3 0-2.8-2.2-3.1-4.2-3.1h-4.2v6.4z" fill-rule="evenodd" /></g></svg>
```

Comment:
why do the svgs have to be embedded in the code?

Reply:
faster loading, if I remember

Reply:
inline svgs get bundled with webpack instead of being a separate request from the public/ directory. also, for posterity: you can manually edit colors and use global vars in the fillRules if necessary

Reply:
ok

Reply:
yea the bundling will decrease load times cause it's not another request. it is a bit more work for the scripts loader and main thread, but on the order of nanoseconds vs. extra few ms for public/ svgs. you might be able to abstract them to the assets/images folder if you really want; i think that's bundled by webp. you'll lose customization though